doc/usage/restructuredtext/directives.rst

```diff
@@ -656,6 +656,10 @@ __ http://pygments.org/docs/lexers
    string are included. The ``start-at`` and ``end-at`` options behave in a
    similar way, but the lines containing the matched string are included.
 
+   If ``start-after``/``start-at`` and ``end-before``/``end-at`` have same strings,
+   ``end-before``/``end-at`` will match with the second line that contains the
```

Comment:
This is not incorrect. Regardless of the existence of `start-after` or `start-at`, `end-before` option always skip the first line. On the other hand, `end-at` tries to match the first line always even if `start-*` option given.

I think it would be better to add a tip for the combination of `start-after` and `end-before` instead.